### ISSUE TITLE
Write initial broadcast message explaining E/W -> GovCloud migration

### ISSUE THREAD

**Issue by brittag** [18F/cg-product#238]
In order for us to give maximum proactive notice to our users, we need to write a broadcast message to all our cloud.gov users informing them about our GovCloud migration plans.

Some of the important points to include (not limited to these):
* Why we're doing this.
* What the benefits will be to them.
* What our expected timeline is.
* Explain that they should reeeeally set up their manifests properly (using the Production-Ready Checklist).
* Explain that we'll help them make sure their .apps.cloud.gov URL doesn't go to a 404 page. (Tracked at https://github.com/18F/cg-product/issues/237)
* Customer responsibility (non-GSA/EPA): SSA, Ed, others will need to log in a new way, using MFA. We'll explain to you later how this will work. We'll send you an invitation.

Also include:
* Reach out to us via email, Slack, etc with questions.
* We'll have office hours at x time with the cloud.gov team and all interested customers - so if you have additional questions or complex questions or just want to learn more, come talk to us and ask us your questions.

Generally this initial broadcast message should be very reassuring and give people confidence that they will be supported and this will go fine.

We may need to do additional followup to make sure a key contact at each project has acknowledged this message (maybe checking against our tenants list - https://github.com/18F/cg-product/issues/229), but that would be a task tracked separately.

**Acceptance criteria**

- [ ] We've scheduled internally with the team when the office hours will be.

We've written up and checked content and posted it to the below places:

- [ ] Update post on the site
- [ ] Dashboard message (linking to the update post)
- [ ] Slack message in #cloud-gov-support and #news (linking to the update post)
- [ ] Email to all users (linking to the update post)

**Comment by mogul** [comment]
@brittag, we're suspending all other work on the GovCloud migration support until we have resolved the "fallback MFA" and "improve service reliability" features, but we still wanted to start front-loading our communication about the migration now, particularly if we do it in tandem with the /updates post that we discussed in the #cloud-gov-skyporter channel yesterday. You said you were looking for content-y things to do... Want to put that/this on your list?

**Comment by mogul** [comment]
@nikzei Moving this particular story to the SkyPorter label, FYI.

**Comment by brittag** [comment]
Yes, I can work on this! Before I start, I'd like to check in with somebody to make sure I'm working with the right assumptions/goals/info/format - who might be the right person for that?

**Comment by mogul** [comment]
Probably me + @nikzei?

**Comment by berndverst** [comment]
@brittag @nikzei @mogul does it make sense for us to tackle this together? Maybe we can have a meeting early next week.

Diego wants us to get the word out as soon as possible that a migration will need to happen, essentially a pre-announcement to set expectations. But of course right now only GSA and EPA can migrate (because of the MFA requirement). Actually, any projects being developed by an external vendor can't be migrated either as that vendor wouldn't have MFA at this time.

I would like to be a reviewer on all communications directly sent to customers as I will need to deal with answering questions.

**Comment by mogul** [comment]
Yeah, let's meet early next week.

Additional thoughts: We'll soon need to publish an omnibus product update covering stuff released this quarter, where we’ll talk about new features that are only in GovCloud and then say “oooh, sorry, non-GSA users, you can’t get in there yet, and it’s waiting on us to implement a fallback MFA-enabled IDP”. 

I feel like we should in fact spin this around: “GSA [and hopefully EPA] users can take advantage of these features simply by logging into the GovCloud environment today. If your agency has a SAML IDP, we can get you setup to use it as well, just send us [this info](https://docs.apps.cloud.gov/ops/federated-identity/)! Otherwise, you’ll have to wait until we have another MFA solution for you... We’re working on that now." Because ideally, we **don’t** want them to rely on our MFA IDP if their agency already has a way of providing/enforcing SSO! That way when they lose their @AGENCYNAME.gov address, their access is cut off. (We’ll have a mechanism for confirming that our fallback MFA users still have control of their e-mail address which pings them periodically, but it will be a while until it's done.)

**Comment by nikzei** [comment]
Since our desire to communicate about migration appears to be high, I would be cautious to send out an omnibus, rather than a clear, simple communication about this one thing. My position is to separate the two and send a more inclusive product update afterward, minimizing the likelihood of this message getting lost in a longer list.

We'll send this out Tuesday, 10/11 + make an announcement about this during all hands the same day. @jameshupp is on it. Thanks everyone!

**Comment by berndverst** [comment]
I agree with @nikzei . It's better to have a separate email "Upcoming cloud.gov changes ACTION REQUIRED" or something like that that talks about the migration by itself, as opposed to including that in general updates. Certainly it could be in the omnibus message, but it would be better to announce in a more direct manner first.

**Comment by jameshupp** [comment]
@nikzei One issue is that there's no longer an all-hands on 10/11. There's a difference between email + news post + announcement on the same day, and email + news post one day and then announcement a week later.

After talking with @dlapiduz what we're aiming for here is an email/news post next Tuesday, a Q&A the following Friday, and then an announcement in the 10/18 all-hands instead.

**Comment by jameshupp** [comment]
This took a bit longer to get going than expected with onboarding and general coming to terms with the vocab of cloud.gov going on around it. An outstanding issue is segmenting the audience we want to email. Two areas of concern:

- all East/West users in one go vs. each individual team
- all space managers vs. all users

Emailing individual teams probably means less vagueness in language, which is important since a lot of cloud.gov users only access the platform irregularly. But it's also incredibly time consuming.

Emailing all users risks incongruent approaches to migration within teams ("I think we should do this now" vs "I want to wait until we deploy this other thing first then worry about it"). If we go to space managers only, the team will be getting a single point of direction about migrating. However, some space managers only control access rather than project/product work.

Aside from email, we'll have a couple other messaging channels internally: all-hands and Slack.

All that said, V3 in this doc is where the draft is now for several venues. I think we're about ready to go. https://docs.google.com/document/d/1FxGO-nbuwh9_nPBmWE4un3il0IYhanWVDIpsNqIaNNc/edit#heading=h.dgftkqcjjjxj

**Comment by nikzei** [comment]
This is great but we really need to get a quick communication out right now. It's function is more of an alert than it is a substantive communication.

I say that we use all of this more detailed and nuanced description for our next update. But for now, we really just want to put people on alert as soon as we can.

I'm going to cut and paste some sections from the work that you've done, @jameshupp to create a short alert. Let's try to get this out by tomorrow or else Monday at the latest.

**Comment by jameshupp** [comment]
I'm compiling a list of information to for migration docs. I think some aspects of this are key to this communication, at least in the larger sense. Sorry for exceeding scope. Should I create a new issue in cg-docs for this?

**Comment by brittag** [comment]
@jameshupp Excellent - a list sounds really helpful! I'd support a new issue in cg-docs - that way it can be its own story with its own acceptance criteria checklist. I've been poking at related things in the docs piecemeal, but it would be great to have a story for the complete picture.

**Comment by dlapiduz** [comment]
Lets target Monday noon to send the notification, What do you all think? We need to figure out how to actually send it though...

**Comment by nikzei** [comment]
@jameshupp and I talked about next steps. Message is drafted. @bernd is out sick. We're going to hold off with sending until we know that we are ready to reference the doc page that will contain updates about this migration. We anticipate that this will go out by Wednesday at the latest, assuming Bernd will be back online before then.

**Comment by berndverst** [comment]
@nikzei and I made some more small edits to the messaging (see the Google doc above in the V3 section). It's definitely ready now.

@jbarnicle is pulling the latest list of users and share.

@berndverst will send out the message tomorrow. No additional work required from anyone but Bernd at this time.

**Comment by berndverst** [comment]
@afeld FYI, I will not send out the messaging until after your change https://github.com/18F/cg-docs/pull/480 is merged.

**Comment by berndverst** [comment]
Here is the latest message (based on work from @jameshupp and edits from @nikzei and @berndverst and incorporating other feedback):

Background
------------
If you’re getting this message, you are hosting applications on the Amazon Web Services East/West environment of cloud.gov. Over the coming months, we are going to deprecate this environment in favor of the AWS GovCloud environment.


Using GovCloud will offer a number of benefits and make compliance easier, including letting you use cloud.gov’s future FedRAMP Moderate JAB Provisional ATO as a foundation for your agency ATO process.

Required Action
----------------
Over the coming months you will need to migrate and redeploy your application to the AWS GovCloud environment of cloud.gov. For information, please see the [GovCloud documentation](https://cloud.gov/docs/apps/govcloud/).

Teams at EPA and GSA already have access to the GovCloud environment and may begin their migration at this time. All other tenants will be notified when access to GovCloud is available. Please look out for further updates with detailed migration instructions and a deprecation timeline of the existing environment.

**Comment by nikzei** [comment]
This went out.
